src/dialect/druidDialect.ts

```diff
@@ -114,6 +114,14 @@ export class DruidDialect extends SQLDialect {
     return `ARRAY[${arr.join(',')}]`;
   }
 
+  public ipParse(value: string): string {
+    return `IP_PARSE${value}`;
```

Comment:
I think you need `IP_PARSE(${value})`, there is no guarantee that value will be wrapped in `()`

Reply:
Thanks for the feedback! Updated.